ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -2329,3 +2329,37 @@ def test_tensorflow_unravel_index(
         indices=indices[0],
         dims=dims[0],
     )
+
+# sequence_mask
+@handle_frontend_test(
+    fn_tree="tensorflow.sequence_mask",
+    dtype_and_lens=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
+        num_arrays=1,
+        min_value=1,
+        max_value=5,
+    ),
+    max_len=st.integers(min_value=5, max_value=1000),
```

Comment:
I feel like this test is not exhaustive, maybe consider doing a compositional strategy instead. As I fear hard-coded values like this might not discover hidden cases.
Also don't forget to check different dtypes

Reply:
Plus, both the input and maxlen need to be integers, which is why I didn't explore other dtypes

Reply:
I would suggest giving them a chance with using compositional strategy, unit tests should be designed to discover those edge cases. So they don't haunt us in the future